README.md

```diff
@@ -15,10 +19,10 @@ npm install --save-dev bunchee
 Usage: bunchee [options]
 
 Options:
-  -v, --version     output the version number
-  -w, --watch       watch src files changes
-  -d, --dest <dir>  specify output dest file
-  -h, --help        output usage information
+  -v, --version        output the version number
+  -w, --watch          watch src files changes
+  -o, --output <file>  specify output filename
```

Comment:
an idea, might not be the final version: use `--cjs` `--esm` `--umd` to specify output files